course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -35,42 +68,41 @@ describe('Unit testing PeopleController', function() {
     // DI sanity check
     it('should inject the providers we requested', function() {
         [$controller, $rootScope, $routeParams, courseInstances, $compile,
-         djangoUrl, $httpBackend, $window].forEach(function(thing) {
+         djangoUrl, $httpBackend, $window, $log].forEach(function(thing) {
             expect(thing).not.toBeUndefined();
             expect(thing).not.toBeNull();
         });
     });
 
     describe('$scope setup', function() {
+        beforeEach(function() {
+            controller = $controller('PeopleController', {$scope: scope });
+        });
+
         afterEach(function() {
             // handle the course instance get from setTitle, so we can always
             // assert at the end of a test that there's no pending http calls.
-            $httpBackend.expectGET('/angular/reverse/?djng_url_name=icommons_rest_api_proxy&djng_url_args=api%2Fcourse%2Fv2%2Fcourse_instances%2F1234567890%2F')
-                .respond(200, '');
+            $httpBackend.expectGET(courseInstanceURL).respond(200, '');
```

Comment:
It says "expect (assert) that the next call to the mock `$http` is to the `courseInstanceURL`, and return a 200 with '' for the body when it gets flushed".  The next line explicitly flushes just one pending http call.

And yeah, we're doing this so that we can make those `verifyNoOutstandingExpectation()`/`verifyNoOutstandingRequest()` calls and have them pass.  The former of which makes sure that all the `expect*()` calls have fired, the latter of which makes sure all the `$http` calls have fired.